app/lib/import_export.rb

```diff
@@ -600,90 +332,34 @@ def extract_patients_details(patients, fields)
-    end
-
-    PATIENT_FIELD_TYPES[:booleans].each do |field|
-      patient_details[field] = patient[field] || false if fields.include?(field)
-    end
-
-    PATIENT_FIELD_TYPES[:phones].each do |field|
-      patient_details[field] = format_phone_number(patient[field]) if fields.include?(field)
-    end
-
-    PATIENT_RACE_FIELDS.each { |race| patient_details[race] = patient[race] || false } if fields.include?(:race)
-
-    patient_details[:name] = patient.displayed_name if fields.include?(:name)
-    patient_details[:age] = patient.calc_current_age if fields.include?(:age)
-    patient_details[:workflow] = patient[:isolation] ? 'Isolation' : 'Exposure'
-    patient_details[:symptom_onset_defined_by] = patient[:user_defined_symptom_onset] ? 'User' : 'System'
-    patient_details[:monitoring_status] = patient[:monitoring] ? 'Actively Monitoring' : 'Not Monitoring'
-    patient_details[:end_of_monitoring] = patient.end_of_monitoring || '' if fields.include?(:end_of_monitoring)
-    patient_details[:expected_purge_ts] = patient.expected_purge_date_exp || '' if fields.include?(:expected_purge_ts)
-    patient_details[:full_status] = patient.status&.to_s&.humanize&.downcase || '' if fields.include?(:full_status)
-    patient_details[:status] = patient.status&.to_s&.humanize&.downcase&.gsub('exposure ', '')&.gsub('isolation ', '') || '' if fields.include?(:status)
-
-    patient_details
-  end
-
-  # Gets a hash of the latest transfers of each patient
-  def get_patients_transfers(patients)
-    transfers = Transfer.latest_transfers(patients)
-    jurisdictions = Jurisdiction.find(transfers.pluck(:from_jurisdiction_id, :to_jurisdiction_id).flatten.uniq)
-    jurisdiction_paths = Hash[jurisdictions.pluck(:id, :path).map { |id, path| [id, path] }]
-    Hash[transfers.pluck(:patient_id, :created_at, :from_jurisdiction_id, :to_jurisdiction_id)
-                  .map do |patient_id, created_at, from_jurisdiction_id, to_jurisdiction_id|
-                    [patient_id, {
-                      transferred_at: created_at.rfc2822,
-                      transferred_from: jurisdiction_paths[from_jurisdiction_id],
-                      transferred_to: jurisdiction_paths[to_jurisdiction_id]
-                    }]
-                  end
-        ]
-  end
-
-  # Gets a hash of the 2 latest laboratories of each patient
-  def get_patients_laboratories(patients)
-    Laboratory.where(patient_id: patients.pluck(:id)).order(specimen_collection: :desc).group_by(&:patient_id).transform_values { |v| v.take(2) }
-  end
-
-  # Extracts assessment data values given relevant fields
+  # Extract assessment data values given relevant fields
   def extract_assessments_details(patients_identifiers, assessments, fields)
     if fields.include?(:symptoms)
-      conditions = ReportedCondition.where(assessment_id: assessments.pluck(:id))
-      symptoms = Symptom.where(condition_id: conditions&.pluck(:id)).order(:label)
```

Comment:
We are no longer ordering by label, was that deemed unnecessary?

Reply:
the order of symptoms here shouldn't matter because they're being placed into a hash, only the order of the fields in `field_data` should matter